### ISSUE TITLE
alignment broke

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#325]
The card alignment in the fixed portion of the chat no longer works.

My guess is adding commas broke it. The easiest fix would be to only use commas in the new display, although it may be pretty simple just to take the commas into account. Either way, it should be trivial to fix.

**Comment by ctm** [comment]
So, after futzing around a little, I decided to make it so ChipCount only adds commas when the alternate option in Display is used, i.e. ":#".

However, I still saw alignment issues when testing, although I was testing on NLHE which uses a big blind ante and perhaps that is a separate bug.

FWIW, I did try adjusting the width to take the commas into consideration and still ran into alignment issues, but at the time I didn't realize there was at least one pre-existing alignment problem.

So, I'm not at all convinced that I've chosen the right tack, but I'm going to remove the "high priority" tag and take this off the Daily Tournament milestone and address it later.
